### ISSUE TITLE
Allow changing language

### ISSUE THREAD

**Issue by gndelia** [hemilabs/ui-monorepo#35]
After #32 , we need to allow users changing languages.

**Comment by qgatssdev** [comment]
@gndelia gm, what's the intention for this feature? What is the intended UX and where should the modal or dropdown be located?

**Comment by gndelia** [comment]
@qgatssdev it hasn't yet been defined. If you want to work on the repo, I'd recommend taking a look at bugs or those "good first issue"

**Comment by gndelia** [comment]
Closing in favor of https://github.com/hemilabs/ui-monorepo/issues/1203
